static_src/components/breadcrumbs.jsx

```diff
@@ -51,7 +51,7 @@ export default class Breadcrumbs extends React.Component {
 
     // Add the parent states
     if (org && org.name) {
-      breadcrumbs.push(<BreadcrumbsItem key="home" url="/#/">overview</BreadcrumbsItem>);
+      breadcrumbs.push(<BreadcrumbsItem key="home" url="/#/">Overview</BreadcrumbsItem>);
```

Comment:
I think we want this lowercase since orgs/spaces/apps are generally lowercase. Otherwise it will stand out too much.